src/app/models/workspace.server.model.ts

```diff
@@ -91,6 +91,18 @@ const compileWorkspaceModel = (db: Mongoose): IWorkspaceModel => {
     return deleted.deletedCount == 1
   }
 
+  WorkspaceSchema.statics.removeFormIdsFromAllWorkspaces = async function ({
+    admin,
+    formIds,
+    session,
+  }: {
+    admin: IUserSchema['_id']
+    formIds: IFormSchema['_id'][]
+    session?: ClientSession
+  }) {
+    await this.updateMany({ admin }, { $pullAll: { formIds } }, { session })
```

Comment:
why not `return` this so that the caller gets back information about the operation?